src/sentry/integrations/slack/views/__init__.py

```diff
@@ -19,7 +21,7 @@ def never_cache(view_func: EndpointFunc) -> EndpointFunc:
 
 def build_linking_url(endpoint: str, **kwargs: Any) -> str:
     """TODO(mgaeta): Remove cast once sentry/utils/http.py is typed."""
-    url: str = absolute_uri(reverse(endpoint, kwargs={"signed_params": sign(**kwargs)}))
+    url: str = absolute_uri(reverse(endpoint, kwargs={"signed_params": sign(salt=SALT, **kwargs)}))
```

Comment:
Is there any chance of this causing requests to fail if they happen mid-deployment? That is, a linking URL gets emitted signed by the new salt, but the webhook request lands on a host that's still trying to unsign it with the old salt, or vice versa.

Reply:
You are absolutely right. There are two cases:
1. Sign with the old salt, unsign with the new salt -- covered by previous PR https://github.com/getsentry/sentry/pull/74839 (compatibility during transitional period)
2. Sign with the new salt, unsign with the old salt -- however, this one would fail during deployments. I have fixed the code so that all hosts will be ready to unsign with the new one, but don't sign payloads with new salt just yet. I will add new salt to `sign` function in the follow-up PR.